README_EN.md

```
@@ -0,0 +1,202 @@
+# 🏠 Care Copilot
+### M5Stack Fire × M5Unified Elderly Care IoT System
```

markdownlint-cli2 (0.17.2)

Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)